draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -242,7 +243,7 @@ informative:
 
 This specification defines a Uniform Resource Name namespace for
 UUIDs (Universally Unique IDentifiers), also known as GUIDs (Globally
-Unique IDentifiers).  A UUID is 128 bits long, and can guarantee
+Unique IDentifiers).  A UUID is 128 bits long, and is intended to guarantee
```

Comment:
I think it's important to not over-promise here - we can't actually guarantee uniqueness, but we can get delivery a high probability. I think the wording should reflect that.